### DISCUSSION TITLE
Como executar o DeepFace no meu PC?

### DISCUSSION THREAD

**Discussion by laggopontes** [community/community#39782] [New to GitHub]
Amigos por gentileza me ajude nesse tema, baixei o aplicativo DeepFace, porem não consigo executa-lo, pois na pasta não há o arquivoo .exe, com faço
para executar ou abrir o app DeepFace já que a linguagem é Python.

**Comment by ossanna16** [comment]
Hi @laggopontes,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on DeepFace. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
